server/server_test.go

```diff
@@ -458,12 +476,17 @@ func newPeer() (*ipfslite.Peer, error) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	log.SetLogLevel("*", "warn")
+	log.SetLogLevel("*", "debug")
+
+	// Bootstrappers are using 1024 keys. See:
+	// https://github.com/ipfs/infra/issues/378
+	corecrypto.MinRsaKeyBits = 1024
```

Comment:
Added this to be consistent with the IPFS-Lite-lite example that works.

Reply:
Ah nice, this is the key issue.